.github/workflows/build.yaml

```diff
@@ -325,7 +334,6 @@ jobs:
       CCACHE_COMPRESS: 'true'
       CCACHE_MAXSIZE: '400MB'
       QPID_SYSTEM_TEST_TIMEOUT: 300
-      QPID_SYSTEM_TEST_SKIP_FALLBACK_SWITCHOVER_TEST: True
```

Comment:
todo, did not mean to delete this here, although this env var is obsolete, for previously deleted test

Reply:
fixed